### ISSUE TITLE
Representativeness of Copernicus data vs. years

### ISSUE THREAD

**Issue by KathiSchleidt** [FAIRiCUBE/data-requests#54]
When a Copernicus dataset claims to be valid for a specific year, what does this really mean? 
- Valid for the entire year?
- A snapshot of (when)?
- ???

Should be clarified with Copernicus responsibles

**Comment by pebau** [comment]
indeed, when the data - which we rely upon - get entered then the data requester should know such details. In the end, whatever they want to do with the data, this will heavily affect what they can do - up to the point where data might actually useless.

**Comment by KathiSchleidt** [comment]
@robknapen what's your take on this issue?

**Comment by pebau** [comment]
> When a Copernicus dataset claims to be valid for a specific year, what does this really mean?
> 
>     * Valid for the entire year?
> 
>     * A snapshot of (when)?
> 
>     * ???

This has been clarified in a telecon together with you; as you moderated, did you also take minutes? For the records, the result was: period of validity of a timeslice = 1 year, distance = 3 years (but may vary and be longer).

This is our working basis, status is as communicated:

- the data are available and and can be used
- re insight: we initially had modelled them through time but then found out that this is so particular that we will enhance calendar functionality, which is under way
- the data are available and and can be used
- until the calendar enhancement is available we will remodel the time axis to be integer for the years, making it even simpler to access, but
- the data are available and and can be used

**Comment by pebau** [comment]
refinement:
period of validity of a timeslice = 1 year (the year in which it has been issued)

**Comment by KathiSchleidt** [comment]
@pebau minor linguistic correction - I'd say the year for which it has been calculated, not when it was issued (maybe years later)

**Comment by KathiSchleidt** [comment]
@pebau can we agree that we use the year for which the dataset was calculated, not some later year when the data was finally published or updated?

Validity is a bit of an issue with Copernicus datasets, as they're often generated from data stemming from multiple previous years. While we can spend another year philosophizing on this, for the present, I believe we should rely on what's stated in the dataset metadata, provide the data as valid for the year indicated in the dataset.

I'd like to see how this can be done correctly, the current approach of making the 2012 dataset valid only from 31.12.11T12:00 - 01.02.11T12:00 (so the 24 hours around New Years Eve) is creative, but not valid

**Comment by pebau** [comment]
@Mohinem you need to fix the year to integer really soon now, Kathi continues getting confused by the current setup.
 
> I'd like to see how this can be done correctly, the current approach of making the 2012 dataset valid only from 31.12.11T12:00 - 01.02.11T12:00 (so the 24 hours around New Years Eve) is creative, but not valid

**Comment by KathiSchleidt** [comment]
@pebau thanks for the insult, and yes, I'm confused by the incorrect provision of years here! ;)
